classes/article/SubmissionFileDAO.inc.php

```diff
@@ -64,6 +64,8 @@ function getGenreCategoryMapping() {
 	function fromRow($row) {
 		if (isset($row['artwork_file_id']) && is_numeric($row['artwork_file_id'])) {
 			return parent::fromRow($row, 'SubmissionArtworkFile');
+		} elseif (isset($row['supplementary_file_id']) && is_numeric($row['supplementary_file_id'])) {
```

Comment:
Supplementary file in the OJS 3.0 sense would mean something that's got a full Dublin Core metadata set, i.e. is indexed separately from the article's metadata.